_includes/assets/sass/_accessibility.scss

```diff
@@ -0,0 +1,41 @@
+/* ==========================================================================
+ACCESSIBILITY
```

Comment:
I think we might be better off here importing the visually-hidden utility calss from the govuk frontend because:
1. It's tested to a high standard 
2. Used widely
3. Maintained by not us ;)

https://github.com/alphagov/govuk-frontend/blob/899264cd89559d31f0a790682254d94ecf66418b/package/govuk/helpers/_visually-hidden.scss

Reply:
This sounds great and the logical thing to do. I've cleared this out with a view to importing givuk frontend.